docs: Homepage and Getting Started section (#5772)
Flagsmith/flagsmith: docs/docs/getting-started/quick-start.md

---
title: Quick Start
sidebar_label: Quick Start
sidebar_position: 2
---

import CodeBlock from '@theme/CodeBlock'; import { JsVersion } from '@site/src/components/SdkVersions.js';

# Flagsmith Quick Start Guide

This tutorial will guide you through the core concepts of integrating Flagsmith into your application.

## Learning objectives

By the end of this tutorial, you will be able to:

> - Create a Project and your first Flag in the Flagsmith dashboard.

Reviewer: As per other conversations, please refrain from capitalising terms that don't need it. 

```suggestion
- Create a Project and your first flag in the Flagsmith dashboard.
```

> - Import the Flagsmith Javascript SDK into your web page.

Reviewer: ```suggestion
- Import the Flagsmith JavaScript SDK into your web page.
```

https://stackoverflow.com/questions/3989731/is-it-javascript-or-javascript

- Connect to the Flagsmith API and retrieve your flags.
- Update your application's behaviour based on a flag's value.

## Prerequisites

You will need access to a Flagsmith instance (either using the SaaS platform or self-hosted) and a basic understanding of HTML and JavaScript.

:::tip

> If you're using the SaaS platform, log in at [Flagsmith.com](https://flagsmith.com/). If you're self-hosting, log in to your deployed instance.

Reviewer: ```suggestion
If you're using the SaaS platform, log in at [app.flagsmith.com](https://app.flagsmith.com/). If you're self-hosting, log in to your deployed instance.
```

:::

## 1. Create a Project and Flag in the Dashboard

> We are going to create an Organisation (if you have not already), a Project, and your first Flag.

Reviewer: ```suggestion
We are going to create an organisation (if you haven't already), a project, and your first flag.
```
